lib/grandpa/grandpa.go

```diff
@@ -550,6 +552,7 @@ func (s *Service) sendVoteMessage(stage Subround, msg *VoteMessage, roundComplet
 	ticker := time.NewTicker(s.interval * 4)
 	defer ticker.Stop()
 
+	// TODO: Send the message just once?
```

Comment:
maybe we can leave this as-is, since the cache should prevent double sends to the same peer